exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java

```diff
@@ -128,27 +121,29 @@ public void submitValidTransaction(TestContext context) {
     post(ApiController.SUBMIT_TRANSACTION_PATH)
         .sendJsonObject(
             new JsonObject(messageJson),
-            context.asyncAssertSuccess(
-                r -> {
+            context.succeeding(
+                r -> context.verify(() -> {
                   // Check the response status
                   int statusCode = r.statusCode();
-                  context.assertEquals(HTTP_OK, statusCode);
+                  assertEquals(HTTP_OK, statusCode);
```

Comment:
`assertThat`

Reply:
I'd keep as is at the moment — it's not the aim of this PR to migrate all usages of `assertEquals` to anything else.